docs/core/dependency-loading/collect-details.md

```diff
@@ -0,0 +1,164 @@
+
+## Prerequisites
+
+- [.NET 5.0 SDK](https://dotnet.microsoft.com/download) or later versions
+- [dotnet-trace](../diagnostics/dotnet-trace.md) tool.
+
+## Collect a trace with assembly loading events
+
+To enable assembly loading events in the runtime and collect a trace of them, use `dotnet-trace` with the following command:
+
+```console
+dotnet-trace collect --providers Microsoft-Windows-DotNETRuntime:4 --process-id <pid>
+```
+
+This will collect a trace of the specified `<pid>`, enabling the `AssemblyLoader` events in the `Microsoft-Windows-DotNETRuntime` provider. The result is a `.nettrace` file.
+
+## View a trace
+
+The collected trace file can be viewed on Windows using the Events view in [PerfView](https://github.com/microsoft/perfview). All the assembly loading events will be prefixed with `Microsoft-Windows-DotNETRuntime/AssemblyLoader`.
+
+## Example (on Windows)
+
+This example uses the [assembly loading extension points sample](https://github.com/dotnet/samples/tree/master/core/extensions/AssemblyLoading). The application attempts to load an assembly `MyLibrary` - an assembly that is not referenced by the application and thus requires handling in an assembly loading extension point to be successfully loaded.
+
+### Collect the trace
+
+1. Navigate to the directory with the downloaded sample. Build the application with:
+
+    ```console
+    dotnet build
+    ```
+
+1. Launch the application with arguments indicating that it should pause, waiting for a key press. On resuming, it will attempt to load the assembly in the default `AssemblyLoadContext` - without the handling necessary for a successful load. Navigate to the output directory and run:
+
+    ```console
+    AssemblyLoading /d default
+    ```
+
+1. Find the application's process ID.
+
+    ```console
+    dotnet-trace ps
+    ```
+
+    The output will list the availabe processes. For example:
+
+    ```console
+    35832 AssemblyLoading C:\samples\core\extensions\AssemblyLoading\AssemblyLoading\bin\Debug\net5.0\AssemblyLoading.exe
+    ```
+
+1. Attach `dotnet-trace` to the running application.
+
+    ```console
+    dotnet-trace collect --providers Microsoft-Windows-DotNETRuntime:4 --process-id 35832
+    ```
+
+1. In the window running the application, press any key to let the program continue. Tracing will automatically stop once the application exits.
+
+### View the trace
+
+Open the collected trace in [PerfView](https://github.com/microsoft/perfview) and open the Events view. Filter the events list to `Microsoft-Windows-DotNETRuntime/AssemblyLoader` events.
+
+![PerfView assembly loader filter image](media/collect-details/assembly-loader-filter.png)
```

Comment:
Please review all of your images. You should use the new `:::image` syntax. This automatically adds a grey border around the image as required. Also, make sure your images are <= 1200 pixels. If you can't fit the content in that size, try using a 100% zoom level on your desktop.
```suggestion
:::image type="content" source="media/collect-details/assembly-loader-filter.png" alt-text="PerfView assembly loader filter image.":::
```